File: TopCoder07/Java/Main.java

public class Main {

	public static void main(String[] args) {
		int n = 2;
		int east = 25;
		int west = 25;
		int south = 25;
		int north = 25;
		CrazyBot cb = new CrazyBot();
		System.out.print(cb.getProbability(n, east, west, south, north));
	}

}

File: TopCoder07/Scala/Main.scala
object Main {

  def main(args: Array[String]): Unit = {
    val n = 2
    val east = 25
    val west = 25
    val south = 25
    val north = 25
    val cb = new CrazyBot
    println(cb.getProbability(n, east, west, south, north))
  }

}